### ISSUE TITLE
Non Secure Web Pages Don't Have a Prompt 

### ISSUE THREAD

**Issue by flikites** [imperviousinc/beacon#3]
This request is for a prompt to appear and be accepted/ignored before loading an unsecured website through beacon. 

![Screenshot_20220426-071805](https://user-images.githubusercontent.com/72367027/165308885-e9cf31a2-4b83-4720-a60c-c725b456d500.png)

**Comment by buffrr** [comment]
Cool suggestion thanks! This would be similar to HTTPS-only mode in firefox correct?

**Comment by FliKites** [comment]
Correct. I think that is the best implementation. Force HTTPS, and allow users to disable the functionality if they wish to do so.
